tensorbay/utility/attr.py

```diff
@@ -61,12 +61,19 @@ class AttrsMixin:
     _attrs_supports: Set[Any]
 
     def __init_subclass__(cls) -> None:
-        cls._attrs_fields = {}
+        attrs_fields = {}
+
+        for base_class in cls.__bases__:
+            base_fields = getattr(base_class, "_attrs_fields", None)
```

Comment:
how about `getattr(base_class, "_attrs_fields", {})`

please check which one is faster

Reply:
`None` is faster than `{}`

a class has the attr, b class not has the attr.
```
In [8]: %timeit get_none(a)
278 ns ± 1.91 ns per loop (mean ± std. dev. of 7 runs, 1000000 loops each)

In [9]: %timeit get_empty_dict(a)
285 ns ± 1.25 ns per loop (mean ± std. dev. of 7 runs, 1000000 loops each)

In [11]: %timeit get_none(b)
161 ns ± 0.501 ns per loop (mean ± std. dev. of 7 runs, 10000000 loops each)

In [12]: %timeit get_empty_dict(b)
265 ns ± 0.591 ns per loop (mean ± std. dev. of 7 runs, 1000000 loops each)
```